### PR TITLE
added PSS DSL syntax

### PR DESCRIPTION
**Pull request by blutsvente**
- [x] I'm the package's author and/or maintainer.
- [x] I have have read [the docs][1].
- [x] I have tagged a release with a [semver][2] version number.
- [x] My package repo has a description and a README describing what it's for and how to use it.
- [x] My package doesn't add context menu entries. *
- [x] My package doesn't add key bindings. **
- [x] Any commands are available via the command palette.
- [x] If my package is a syntax it doesn't also add a color scheme. ***
- [x] I use [.gitattributes][3] to exclude files from the package: images, test files, sublime-project/workspace.

My package is for syntax-highlighting the PSS DSL language.

There are no packages like it in Package Control.

[1]: https://packagecontrol.io/docs/submitting_a_package
[2]: https://semver.org
[3]: https://www.git-scm.com/docs/gitattributes#_export_ignore

### CODE REVIEW

**Comment by braver** [comment]
You could save on bytes stored on user’s systems and sent over the net by excluding files like the PNG’s using gitattributes.

**Comment by blutsvente** [comment]
Thanks for the feedback, all implemented. I have not yet created a new release of the language package, waiting in case more issues are found per this pull-request.

**Comment by braver** [comment]
Thanks, looks good 👍🏻

File: repository/p.json [hunk 1]
```diff
@@ -2719,6 +2719,18 @@
 				}
 			]
 		},
+		{
+			"name": "PSS",
+			"details": "https://github.com/blutsvente/PSS-DSL-syntax",
+			"issues": "https://github.com/blutsvente/PSS-DSL-syntax/issues",
```
Comment (braver) [inline repository/p.json:2725]: That’s not necessary, package control understands normal GitHub urls and will find the issues without you specifying it here.

**Comment by blutsvente** [comment]
Released 1.0.1 including all changes discussed.

**Pull request merged**